sesman/chansrv/chansrv.c

```diff
@@ -39,6 +39,9 @@
 #include "xrdp_sockets.h"
 #include "audin.h"
 
+#define CHANNEL_NAME_BYTES 7
```

Comment:
Isn't this defined in MS specification?

Reply:
You're quite right. There's no name for it, but it's in section 2.2.1.3.4.1 of MS-RDPBCGR. I've added the definition to the include file and referenced that. Let me know if/when you want me to squash.

Reply:
Thanks!